src/main/java/data_access/DataGetterClass.java

```diff
@@ -12,21 +13,20 @@
 public class DataGetterClass {
 
 
-    public static JsonNode getData(String queryUrl) throws IOException {
+    public static JsonNode getData(String queryUrl, CurrentUser currentUser) throws IOException {
         // Construct the query URL
-        String Token = "BQByOEUh6cwcK6P96sT_wFdrMqMGqMH0CEXycNkXsyAKHeZxFSlo9UNt-urG-_iIGgX1yQpqozY22FS2AVoh7E4UND5ASOvbpzjcmnCAFspngnjhKyGAIdvy-3PSOJXs2s1dj5wwckor2NxTd5cZCIVK1mGJwcSR1t_0HeCuZx-ubxliwgYwLcc0pcxAk_EQhKzNW0GmGrVjSkMQm_Lds0vK3bl-Pqc22oBF4vwJcjcOuw";
-
+        final String token = currentUser.getAccessToken();
```

Comment:
Good fix to our access token problem, should ensure that access token is still working for other use cases as well